### DISCUSSION TITLE
Mac compatability

### DISCUSSION THREAD

**Discussion by dokjapp** [drizzle-mizzle/CharacterAI-Discord-Bot#66] [Q&A]
Is there any way to use this on a mac device? I don't have any windows/linux devices

**Comment by drizzle-mizzle** [comment]
I can make a macOS version with a next update

**Reply by drizzle-mizzle** [reply]
Check out the latter release, I ve made a Mac version. Though, it's not tested, at all, and I'm not sure what problems you might encounter.
